.github/workflows/download-image.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Docker Images
+# on:
+#   schedule:
+#     - cron: "*/15 * * * *"
+#   workflow_dispatch:
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+jobs:
+  docker-update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      GCR_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is there a reason to put this in the environment, rather than just referring to `${{ secrets.GITHUB_TOKEN }}` in line 52 below? If not then I'd do that, so that the variable assignment/usage doesn't distract from what's happening.

Reply:
Fixed